wgengine/magicsock/magicsock.go

```diff
@@ -2631,10 +2640,11 @@ func (c *Conn) Rebind() {
 	c.mu.Lock()
 	c.closeAllDerpLocked("rebind")
 	haveKey := !c.privateKey.IsZero()
+	myDerp := c.myDerp
 	c.mu.Unlock()
```

Comment:
Nit: This could all be:

```go
c.mu.Lock()
c.closeAllDerpLocked("rebind")
if !c.privateKey.IsZero() {
  c.startDerpHomeConnectLocked()
}
c.mu.Unlock()
```

(instead of grabbing haveKey and myDerp and unlocking before using them)

Reply:
done